falcon/api.py

```diff
@@ -350,6 +353,9 @@ def add_route(self, uri_template, resource, *args, **kwargs):
         self._router.add_route(uri_template, method_map, resource, *args,
                                **kwargs)
 
+    def add_static_route(self, prefix, folder):
+        self._static_routes.add(prefix, Statics(prefix, folder))
```

Comment:
I think it could be helpful to always insert each new prefix at the head of the list. This would ensure a predictable behavior, which we can (and should) document and include in our tests. With this behavior in place, we can note in the docs that duplicate prefixes will override previous ones, and that a more specific prefix may be added _after_ a more general one (e.g., when "/foo/bar" is added after "/foo", a resource such as "/foo/bar/thing.js" would be mapped to the "/foo/bar" route, while "/foo/tmp/thingtoo.js" would be mapped to the "/foo" route.

```py
self._static_routes.insert(0, Statics(prefix, folder))

Reply:
We should also take the opportunity to verify that the prefix begins with a forward slash, and normalize the string so that it always ends with one as well (so that we match on the complete path segment; matching on the complete segment should also be verified in the unit or functional tests).